test-application/store.fyre

```diff
@@ -3,3 +3,9 @@ use_parent_signal, set_parent_signal, get_parent_signal = create_signal(2)
 
 use_clock_signal, set_clock_signal, _ = create_signal(0)
 ---
+
+<pyml>
+    <div>
+       <slot></slot>
+    </div>
+</pyml>
```

Comment:
`store.fyre` should not have any pyml code

Reply:
By adding pyml in `store.fyre` I was able to raise an exception for if the user for "some reason" forget to add the `<slot>` tag on the component definition.
I will remove it from here and for now the other way I can think of solving this edge case would be adding the inner content of `<slot>` to result html when `<slot>` tag is not found.